Source/Dafny/Resolver.cs

```diff
@@ -15676,7 +15713,7 @@ class MethodCallInformation
       return subst;
     }
 
-    private void ResolveDatatypeValue(ResolveOpts opts, DatatypeValue dtv, DatatypeDecl dt, Type ty) {
+    private bool ResolveDatatypeValue(ResolveOpts opts, DatatypeValue dtv, DatatypeDecl dt, Type ty, bool complain = true) {
```

Comment:
Please add a `///` comment saying what the boolean result value means.

Reply:
Done